### PR TITLE
BROKER: Add Model & Migration

### PR DESCRIPTION
**Pull request by tsatay**

### CODE REVIEW

File: Taarafo.Core/Migrations/20220325004604_AddPostImpressionModel.cs [hunk 1]
```diff
@@ -0,0 +1,55 @@
+﻿using System;
```
Comment (ShriHumrudha) [inline Taarafo.Core/Migrations/20220325004604_AddPostImpressionModel.cs:1]: Add copyrights header

File: Taarafo.Core/Models/PostImpressions/ImpressionType.cs [hunk 2]
```diff
@@ -0,0 +1,14 @@
+﻿// ---------------------------------------------------------------
+// Copyright (c) Coalition of the Good-Hearted Engineers
+// FREE TO USE TO CONNECT THE WORLD
+// ---------------------------------------------------------------
+
+namespace Taarafo.Core.Models.PostImpressions
+{
+    public enum ImpressionType
+    {
+        LIKE,
```
Comment (ShriHumrudha) [inline Taarafo.Core/Models/PostImpressions/ImpressionType.cs:10]: Why all caps?

File: Taarafo.Core/Models/PostImpressions/PostImpression.cs [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+// ---------------------------------------------------------------
+
+using System;
+using Microsoft.EntityFrameworkCore;
+using Taarafo.Core.Models.Posts;
+using Taarafo.Core.Models.Profiles;
+
+namespace Taarafo.Core.Models.PostImpressions
+{
+    [Keyless]
```
Comment (ShriHumrudha) [inline Taarafo.Core/Models/PostImpressions/PostImpression.cs:13]: I would like to learn about this [Keyless]. Please share pointers.

File: Taarafo.Core/Models/PostImpressions/PostImpression.cs [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+    {
+        public Guid PostId { get; set; }
+        public Post Post { get; set; }
+
+        public Guid ProfileId { get; set; }
+        public Profile Profile { get; set; }
+
+        public DateTimeOffset CreatedDate { get; set; }
+        public DateTimeOffset UpdatedDate { get; set; }
+        public ImpressionType ImpressionType { get; set; }
```
Comment (ShriHumrudha) [inline Taarafo.Core/Models/PostImpressions/PostImpression.cs:24]: Move this field above the auditing fields.

**Review by ShriHumrudha — CHANGES_REQUESTED** [review]
Am I missing References.cs file here?

**Comment by tsatay** [comment]
> 

Yes, I have not created References.cs file yet.

File: Taarafo.Core/Models/PostImpressions/PostImpression.cs (hunk 3 above)
Comment (tsatay) [reply Taarafo.Core/Models/PostImpressions/PostImpression.cs:13]: Here it is: https://docs.microsoft.com/en-us/ef/core/modeling/keyless-entity-types?tabs=data-annotations

**Pull request closed without merging**